src/vos/vos_pool.c

```diff
@@ -434,31 +454,17 @@ vos_pool_kill(uuid_t uuid, bool force)
 			vos_pool_decref(pool); /* -1 for lookup */
 			continue;	/* try again */
 		}
+		pool->vp_dying = 1;
 		vos_pool_decref(pool); /* -1 for lookup */
 
-		if (force) {
-			D_ERROR("Open reference exists, force kill\n");
-			/* NB: rebuild might still take refcount vos_pool,
-			 * we don't want to fail destroy or locking space.
-			 */
-			break;
-		}
-		D_ERROR("Open reference exists, cannot kill pool\n");
+		D_ERROR(DF_UUID": Open reference exists, pool deletion is deferred\n",
```

Comment:
Should this always be a D_ERROR() log? For example in tgt_destroy() -> tgt_destroy_cleanup() that runs after this, it produces a D_INFO(). Should both locations be D_INFO()?

Reply:
Yeah, maybe I'd change it to D_WARN, I just don't want this message being suppressed in our daily test.

Seems the CI testing is still moving, I'll update it (along with your suggestion of adding a RAS event for this) if the patch needs be refreshed, otherwise, I'd leave the change to a follow-on patch, sounds good to you?

Reply:
Yes sounds good. Reasoning behind RAS notification was that an administrator may then take action by (manually) inspecting that the pool deletion that is deferred eventually completes. e.g., by running a command such as dmg storage scan list-pools